src/map2d/pcd2track.py

```diff
@@ -0,0 +1,61 @@
+import numpy as np
+
+class TrackMaker:
+    """
+    todo:
+        - detect empty pcd sector?
+        - how to connect pcd's?
+
+    Class to convert point cloud data to track data.
+    From 2d batch of points we are calculating 2 matrices:
+    - horizontal gradient
+    - vertical gradient
+    Then setting magnitude of gradient as a square root of sum of squares of both gradients.
+    Based on that magnitude we are setting threshold to get track data,
+    where łazik can drive on.
+    0 - can't drive
+    1 - can drive
+    """
+    def __init__(self):
+        self.visualize = False # todo: implement visualization
+        self.gradient_magnitude = None
+
+        if self.visualize:
+            self.vis = "todo: implement visualization"
+    
+    def __del__(self):
+        if self.visualize:
+            self.vis.destroy_window()
+
```

Comment:
na szubko pomysł jest taki z racji że to już mapy "2d"  to użyć plt i aktualizować je w sposób rzeczywisty  
```python
    def visualize_gradients(self):
        self.axes[0, 0].imshow(self.Gx, cmap='viridis')
        self.axes[0, 0].set_title('Horizontal Gradient Gx')
        self.axes[0, 1].imshow(self.Gy, cmap='viridis')
        self.axes[0, 1].set_title('Vertical Gradient Gy')
        self.axes[1, 0].imshow(self.gradient_magnitude, cmap='viridis')
        self.axes[1, 0].set_title('Gradient Magnitude')
        self.fig.canvas.draw()  # Update the figure

    def visualize_threshold(self, thresholded):
        self.axes[1, 1].clear()  # Clear the previous threshold image
        self.axes[1, 1].imshow(thresholded, cmap='binary')
        self.axes[1, 1].set_title('Thresholded Gradient Magnitude')
        self.fig.canvas.draw()  # Update the figure
```